packages/core/src/child-proxy/ChildProcessProxyWorker.ts

```diff
@@ -76,7 +76,7 @@ export default class ChildProcessProxyWorker {
     }
   }
 
-  private doCall(message: CallMessage): {} | PromiseLike<{}> | undefined {
+  private doCall(message: CallMessage): Record<string, any> | PromiseLike<Record<string, any>> | undefined {
```

Comment:
I think al lot of the `Record<string, any>` things could actually be `Record<string, unknown>`. That way we don't accidentally use it when we should narrowed the type with a type guard.